documentation/asciidoc/computers/linux_kernel/building.adoc

```diff
@@ -104,79 +134,70 @@ $ make bcm2711_defconfig
 | Raspberry Pi 400
 | Raspberry Pi Compute Module 4S
 | Raspberry Pi Compute Module 4
+|===
 
+[NOTE]
+====
+The 32-bit distribution of Raspberry Pi OS on a Raspberry Pi 4B, 5, 400, Compute Module 4, or Compute Module 4S uses a 32-bit userland, but a _64-bit kernel_. To build a 32-bit kernel, set `ARCH=arm`. To boot a 32-bit kernel, set `arm_64bit=0` in `config.txt`.
+====
 
-.10+^.^| 64-bit
-| Raspberry Pi 3
-.9+.^|
-[source,console]
+[[native-customisation]]
+==== Customise the kernel version using `LOCALVERSION`
+
+Adjust `LOCALVERSION` to clarify that you are running your own kernel in the output of `uname` and prevent the kernel from overwriting existing modules in `/lib/modules`.
```

Comment:
Only a minor thing, but it might be worth swapping round the two parts of the sentence, as IMHO not overwriting existing kernel modules (which is what the `make modules_install` part does) is probably more important than the `uname` output-string?